distributor-api-v1/changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 08.03.2021
+
+* Added `AverageAmountPerTimeUnit` field to [Room price](./operations.md#room-price) replacing the `AverageAmountPerNight`
+* Added `ChargingMode` field to [Product](./operations.md#product) replacing the `Charging`
+* Added `PostingMode` field to [Product](./operations.md#product) replacing the `Posting`
+* Added `SettlementMaximumTimeUnits` field to [Rate group](./operations.md#rate-group) replacing the `SettlementMaximumNights`
+
 ## 21.07.2021
```

Comment:
also I realized I forgot to update the date when the PR was merged so we have the creation date in the changelog, should I change it to 08.02.2021 now? @koprivajakub